### PR TITLE
Wmm/making player class

### PR DESCRIPTION
**Pull request by williammcintosh**
Finished moving over class variables and methods. Converted all `generalChannel.send` statements as returns. All mentioned of `arguments` were renamed to `prompts` since functions inside of classes are constrained to JS' strict rules.

There are still some areas that need to be completed but I'm stuck until the other classes are finished:

1. Every reference to `myList` needs to be a request to the future `BoardManager`.
2. `drawCard()` need to be a request to some future manager.
3. `getRandomInt()` is currently in the Player class but needs to move elsewhere.
4. Every reference to `playerList` needs to be a request to the future `BoardManager`.
5. The MoBot does not in anyway interact with the Player class. MoBot will be converted to something else in the future, but it’s important to recognize that there are no outside interactions with the Player class yet.
6. We may need to determine how `DeedCommand()` handles the `receivedMessage` argument or if the Player even needs to handle it since the request goes back to the future `BoardManager`.
7. In the `BailCommand` method, the Player class probably should’ve have access to the Community Chest or Chance decks. The lines `comDiscard.push` and `chanceDiscard.push` probably needs to be handled by another, future class.

### CODE REVIEW

File: assets/save.json
Comment (Iwasson) [inline assets/save.json:1]: Can you drop this file. We do not want to commit or upload any save files as that would be generated on runtime.

File: monopoly/MoBot.js
Comment (Iwasson) [inline monopoly/MoBot.js:1]: Can you drop this change. It looks like you ran a linter here which is causing a crazy diff. These kinds of diffs are incredibly hard to read and this change should be included in a different PR.

**Comment by Iwasson** [comment]
Only other comment, can you squash these commits into one before merge

File: monopoly/MoBot.js
Comment (williammcintosh) [reply monopoly/MoBot.js:1]: I think I got it!

File: assets/save.json
Comment (williammcintosh) [reply assets/save.json:1]: I think I did it!
